mail/mailutils/Portfile

```diff
@@ -0,0 +1,96 @@
+revision            0
+
+categories          mail
+platforms           darwin
+license             GLP-3
+
+maintainers         nomaintainer
+
+description         General-Purpose Mail Packages
+long_description    Mailutils is a swiss army knife of electronic mail handling. \
+                    It offers a rich set of utilities and daemons for processing e-mail.
+
+homepage            https://mailutils.org/
+master_sites        gnu:mailutils
+use_xz              yes
+
+checksums           rmd160          6f222222b6337afe6014d6af16d6e4bcf6884009 \
+                    sha256          46e23b53caba26b55c1dfd4e012047742bf6d39434c72d4da7ac4d9bf9fdf01e \
+                    size            2813412
+
+configure.args      --without-readline \
+                    --with-ldap=no \
+                    --enable-build-servers=no \
+                    --with-guile=no \
+                    --without-gnutls \
+                    --without-fribidi \
+                    --with-tcp-wrappers=no \
+                    --disable-cxx \
+                    --disable-python
+
+depends_build       port:coreutils \
+                    port:gawk \
+                    port:grep \
+                    port:gsed \
+                    port:m4 \
+
+variant readline description {Enable readline} {
+    configure.args-delete   --without-readline
+    depends_lib-append      port:readline
+}
+
+variant ldap description {Build with ldap support} {
+    configure.args-delete   --with-ldap=no
+}
+
+variant server description {Build servers} {
+    configure.args-delete   --enable-build-servers=no
+    configure.args-append   --enable-build-servers
+}
+
+variant guile description {Enable support for Guile scripting} {
+    configure.args-delete   --with-guile=no
+    depends_lib-append      port:guile
+}
+
+variant gnutls description {Build with GNU TLS support} {
+    configure.args-delete   --without-gnutls
+    depends_lib-append      port:gnutls
+}
+
+variant fribidi description {Build with fribidi support} {
+    configure.args-delete   --without-fribidi
+    depends_lib-append      port:fribidi
+}
+
+variant tcp_wrappers description {Build with tcp_wrappers support} {
+    configure.args-delete   --with-tcp-wrappers=no
+    depends_build-append    port:tcp_wrappers
+}
+
+variant python27 description {Enable support for Python 2.7 scripting} {
+    configure.args-delete   --disable-python
+    configure.args-append   --enable-python
+    configure.python        ${prefix}/bin/python2.7
+
+    depends_lib-append      port:python27
+}
+
+variant ndbm conflicts gdbm description {Build with NBDM support} {
+    configure.args-append   --with-ndbm
+}
+
+variant gdbm conflicts ndbm description {Build with GNU DBM support} {
+    configure.args-delete   --with-ndbm
+    configure.args-append   --with-gdbm
+
+    depends_lib-append      port:gdbm
+}
+
+default_variants -gdbm -server -python27 -readline +ldap -servers -guile -gnutls -freebidi +tcp_wrappers
```

Comment:
We don't specify the negative `default_variants`. Only list the positive (i.e. selected by default) variants.

Reply:
fixed. thanks.